CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## Unreleased
+## 0.13.1 - 2020-03-20
```

Comment:
this should be changed
back to "unreleased" until we make a release PR?

Reply:
ok